radio/src/datastructs_private.h

```diff
@@ -887,6 +887,9 @@ PACK(struct RadioData {
   NOBACKUP(uint8_t  imperial:1);
   NOBACKUP(uint8_t  disableRssiPoweroffAlarm:1);
   NOBACKUP(uint8_t  USBMode:2);
+  #if defined(STICK_DEAD_ZONE)
+  NOBACKUP(uint8_t  stickDeadZone:3);
```

Comment:
@pfeerick you cannot add 3 bits just like that ;-) you need to make sure the 5 others are somewhere, otherwise the generator won’t see them.

Reply:
Thanks. Fixed, and also added the check for the struct size as that was missing for the NV14